### ISSUE TITLE
Building with Symfony 4.4 results in error using split function

### ISSUE THREAD

**Issue by TwoLeaves** [sandreas/m4b-tool#90]
Splitting an m4b file without conversion produces an error when m4b-tool is compiled against Symfony 4.4, as Command::execute must now always return an integer value. I made a pull request for this SplitCommand case, though it seems execute is used elsewhere in the codebase also (I would have made a pull request adjusting all the uses of execute were I confident it was a sane fix).

```
$ m4b-tool split --no-conversion book.m4b 
an error occured, that has not been caught:
Array
(
    [type] => 16384
    [message] => Return value of "M4bTool\Command\SplitCommand::execute()" should always be of the type int since Symfony 4.4, NULL returned.
    [file] => phar:///usr/bin/m4b-tool/vendor/symfony/console/Command/Command.php
    [line] => 258
)
```

**Comment by sandreas** [comment]
Is this one fixed with your pull request?

**Comment by TwoLeaves** [comment]
Yes it seems so. Though the other uses of execute might be problematic; I have not checked. Thank you.

**Comment by sandreas** [comment]
> Though the other uses of execute might be problematic; I have not checked. 
Good point. I'll try to check this in the next days.
